src/autorest-core/resources/default-configuration.md

```diff
@@ -70,6 +70,11 @@ use-extension:
   "@microsoft.azure/autorest.typescript": "~2.0.0"
 ```
 
+``` yaml $(model-validator)
```

Comment:
That infers that in order to enable it people should say `--model-validator` on the command line.

👍

Reply:
yes, or model-validator true in their readme.md file, are there any docs that should be updated to indicate this?